qutebrowser/config/configdata.yml

```
@@ -1794,6 +1794,14 @@ input.media_keys:
 
     On Linux, disabling this also disables Chromium's MPRIS integration.
 
+input.match_counts:
+  default: true
+  type: Bool
+  desc: >-
+    This can be used for emacs-like bindings, so that number keys are passed
+    through according to `input.forward_unbound_keys` rather than matched as a
+    count.
```

Do you mind changing this to imperative mood to better fit with the existing boolean option (see 494aceec451e307b0227b0fdd0f86d419dbc11d1)?

---

Not at all, thanks for pointing that out.

How about this?
```suggestion
    Match number keys as counts.

    When disabled this can be used for emacs-like bindings, so that number keys
    are passed through according to `input.forward_unbound_keys`.
```

---

To me this feels like it explains the setting by describing what it *doesn't* do.
I'd expect most users of a 'vi-like' browser not being familiar with `emacs-like` bindings so I'd prefer that to come across as the exception it .

How about this instead?

```suggestion
    Interpret number prefixes as counts for bindings.
    
    This enables for vi-like bindings that can be prefixed with a number to
    indicate a count.
    Disabling it allows for emacs-like bindings where number keys are passed
    through (according to `input.forward_unbound_keys`) instead.
```

---

That makes sense! Thanks for elaborating!